common/reviews/api/webpack-plugin-utilities.api.md

```diff
@@ -0,0 +1,24 @@
+## API Report File for "@rushstack/webpack-plugin-utilities"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import type * as Webpack4 from 'webpack4';
```

Comment:
consumers aren't going to have this, so TS will complain. We should probably actually just drop the webpack 4 typings and only import from `'webpack'`. Keep both peer dependencies around, though.